history.txt

```diff
@@ -5,6 +5,21 @@
 OMERO version history
 =====================
 
+5.4.5 (March 2018)
+--------------------
+
+This is a bug-fix release reactivating the thumbnail cache
+inadvertently disabled in 5.4.4 while fixing a pyramid issue.
+
+Improvements include:
+
+-  reactivated thumbnail caching
+-  improved removepyramids help
+-  fixed display of thumbnails when searching for images by ID
+-  increased OMERO.web log size
+-  fixed CLI list subcommand
```

Comment:
@mtbc asked me to change this to "fixed CLI config list subcommand" on the website PR so you should probably update this match. Thanks